sys_conn_df_windows.go

```diff
@@ -13,17 +13,26 @@ import (
 
 const (
 	// same for both IPv4 and IPv6 on Windows
-	// https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Networking/WinSock/constant.IP_DONTFRAG.html
-	// https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Networking/WinSock/constant.IPV6_DONTFRAG.html
-	IP_DONTFRAGMENT = 14
-	IPV6_DONTFRAG   = 14
+	// https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Networking/WinSock/constant.IP_MTU_DISCOVER.html
+	// https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Networking/WinSock/constant.IPV6_MTU_DISCOVER.html
+	IP_MTU_DISCOVER   = 14
+	IPV6_MTU_DISCOVER = 14
+)
+
+// enum PMTUD_STATE from ws2ipdef.h
```

Comment:
Can we link to the source here?

Reply:
@marten-seemann `ws2ipdef.h` in Windows SDK is the source.